dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/protocol/AbstractProtocol.java

```diff
@@ -42,7 +43,7 @@
 /**
  * abstract ProtocolSupport.
  */
-public abstract class AbstractProtocol implements Protocol {
+public abstract class AbstractProtocol implements DelegateExporterMap, Protocol {
```

Comment:
I think it's better to make `DelegateExporterMap` as AbstractProtocol's property than as its super interface.

That is, AbstractProtocol has a `DelegateExporterMap` rather is a `DelegateExporterMap`